### PR TITLE
feat(ios): Supported new property ‘masterViewVisible’ to show and hide master view in splitwindow

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-26657

1. This PR introduces new property 'masterViewVisible', which can be used to show and hide master view.
2. Fixes issues 'masterIsOverlayed' when device is in landscape mode but app is open with another app on screen (iPad split view). 
3. Fixes document issue. In document  in Alloy example 'visible' event is there but this event is removed.

Test Case -
```
var button = Ti.UI.createButton({ title: 'Click' });

var detail = Ti.UI.createWindow({ 
    backgroundColor: 'white',
    leftNavButton: button
});
var label1 = Ti.UI.createLabel({ text: 'Toggle Master View' });
detail.add(label1);
var detailNav = Ti.UI.createNavigationWindow({ window: detail });


var master = Ti.UI.createWindow({ backgroundColor: 'gray' });
var label2 = Ti.UI.createLabel({ text: 'Master View' });
master.add(label2);
var masterNav = Ti.UI.createNavigationWindow({ window: master });

var splitWin = Ti.UI.iOS.createSplitWindow({
    detailView: detailNav,
    masterView: masterNav,
    showMasterInPortrait: true,
    masterIsOverlayed: true,
});
splitWin.open();

button.addEventListener('click', function(e){
    Ti.API.info("Left button clicked");
});

label1.addEventListener('click', function(e){
    splitWin.setMasterViewVisible(!splitWin.masterViewVisible);
});
```

### CODE REVIEW

**Comment by hansemannn** [comment]
I'd love to see a refactored version of this API that supported the native `UISplitViewController` instead of a custom one. This one was made for backwards compatibility with iOS 7 back in the days, but since iOS 8 is the current minimum (and even iOS 9 in SDK 8), it'd be great to work on that approach. ✌️

**Comment by vijaysingh-axway** [comment]
@hansemannn I agree with you. I have created a ticket https://jira.appcelerator.org/browse/TIMOB-26692 for refactoring Ti.UI.iOS.SplitWindow. Thanks!

File: tests/Resources/ti.ui.ios.splitwindow.addontest.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+				backgroundColor: 'yellow'
+			})
+		});
+		should(splitWindow.masterView).be.an.Object;
+		should(splitWindow.detailView).be.an.Object;
+	});
+});
+
+// describe.ios('Titanium.UI.iOS.SplitWindow', function () {
+// TODO Add tests for SplitWindow type!
```
Comment (janvennemann) [inline tests/Resources/ti.ui.ios.splitwindow.addontest.js]: Add a simple test for the default values of properties `showMasterInPortrait`, `masterIsOverlayed` and `masterViewVisible`

**Review by janvennemann — CHANGES_REQUESTED** [review]
Just a minor note for the new unit test. Otherwise the code looks good!

File: tests/Resources/ti.ui.ios.splitwindow.addontest.js (hunk 1 above)
Comment (vijaysingh-axway) [reply tests/Resources/ti.ui.ios.splitwindow.addontest.js]: @janvennemann  If you agree, I would like to add other unit test cases for SplitWindow in TIMOB-26692 while doing refactoring for SplitWindow and using native iOS APIs. Thanks!

**Review by ssjsamir — APPROVED** [review]
**FR Passed** When clicking on toggle MasterView it hides/shows the view.

**Test Environment**
```
ipad pro 12.9 inch (12.1 sim)
APPC CLI: 7.0.0
Operating System Name: Mac OS Mojave
Operating System Version: 10.14.2
Node.js Version: 8.9.1
Xcode 10.1
```

File: tests/Resources/ti.ui.ios.splitwindow.addontest.js (hunk 1 above)
Comment (janvennemann) [reply tests/Resources/ti.ui.ios.splitwindow.addontest.js]: Ok, sounds good to me!

**Review by janvennemann — APPROVED** [review]

**Pull request merged**